spec/unit/pops/types/p_sensitive_type_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'spec_helper'
+require 'puppet/pops'
+require 'puppet_spec/compiler'
+
+module Puppet::Pops
+module Types
+describe 'Sensitive Type' do
+  include PuppetSpec::Compiler
+
+  context 'as a type' do
+    it 'can be created with the type factory' do
+      t = TypeFactory.sensitive
+      expect(t).to be_a(PSensitiveType)
+      expect(t).to eql(PSensitiveType::DEFAULT)
+    end
+
```

Comment:
+ can be created with a parameter, for example `TypeFactory.sensitive(PIntegerType::DEFAULT)`